e2e-tests/alttext.spec.ts

```diff
@@ -113,13 +113,13 @@ test('Alt Text', async ({ page }) => {
   };
   await expect(intersectionProperties.heading).toBeVisible();
   await expect(intersectionProperties.content).toBeVisible();
-  await expect(intersectionProperties.content).toHaveText('The plot is sorted by size. There are 12 non-empty intersections, all of which are shown in the plot. The largest 5 intersections are School Male (3), the empty intersection (3), Just Male (3), Duff_Fan Male Power_Plant (3), and Evil Male (2).');
+  await expect(intersectionProperties.content).toHaveText('The plot is sorted by size. There are 12 non-empty intersections, all of which are shown in the plot. The largest 5 intersections are School, and Male (3), Just Unincluded (3), Just Male (3), Duff Fan, Male, and Power Plant (3), and Evil, and Male (2).');
```

Comment:
Just Unincluded was named Unincluded before. I changed the name 'Unincluded' to 'the empty intersection' while doing alt-txt generation. If you plan to keep 'Just Unincluded' let me know, and I will change my alt-txt code.